Azure/communication-ui-library-android
# [Refactor] Modify Video and Mic buttons on call screen regarding Accessibility

## Purpose
Modify announcement for video and mic buttons on call screen to reflect the status change for the buttons.

## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Join a call with talkback on, navigate to video or mic button to verify if the status being announced. also verify announcement for that after clicking on buttons.
